src/denomination.rs

```diff
@@ -1,6 +1,11 @@
+
```

Comment:
It looks like this file is still excluded from cargo format, there's a few formatting issues in here (extra lines, etc..)